src/core/frames/frame_controller.js

```diff
@@ -276,7 +276,13 @@ export class FrameController {
     return !defaultPrevented
   }
 
-  viewRenderedSnapshot(_snapshot, _isPreview) {}
+  viewRenderedSnapshot({ currentSnapshot: { element: frame } }) {
+    return dispatch("turbo:frame-render", {
+      detail: {},
```

Comment:
This is where we'll be able to assign the `renderMethod` key based on the availability of the `renderer` instance